src/K9OCRS/ClientApp/src/App.js

```diff
@@ -20,6 +23,9 @@ export default class App extends Component {
         <Route path='/fetch-data' component={FetchData} />
         <Route path='/classtypes' component={ClassTypesDashboard} />
         <Route path='/classtype/1' component={ClassTypeSetup} />
+        <Route path='/DogRegistration' component={DogRegistration} />
+        <Route path='/DogProfile' component={DogProfile} />
+        <Route path='/DogList' component={DogList} />
```

Comment:
This is to keep the routing consistent with the Figma design
- The `exact` parameter makes it so the route won't match any sub-routes
- The colon on `path='/MyDogs/:dogId'` means that the value of `dogId` will be provided on the url and we can later use it to load the data of that specific dog
```suggestion
        <Route path='/DogSetup' component={DogSetup} exact />
        <Route path='/MyDogs/:dogId' component={DogDetails} />
        <Route path='/MyDogs' component={MyDogs} exact />
```